src/scrollTo.js

```diff
@@ -189,7 +189,10 @@ export const scroller = () => {
             const containerBottom = containerTop + container.offsetHeight;
             const elementTop = targetY;
             const elementBottom = elementTop + element.offsetHeight;
-            if (elementTop >= containerTop && elementBottom <= containerBottom) {
+            if (
+                elementTop >= containerTop && elementBottom <= containerBottom
+            ) {
+                onDone(element);
```

Comment:
This is all I changed, I suppose `prettier` did the above formatting change.

I decided not to try to re-use the existing `done` method because most of what's in there is not applicable to this path of execution:
* We don't want to reset any positioning using `topLeft`
* We've never called `step`, so `timeStart` is still undefined and doesn't need to be reset
* We have not wired any events, so `_.off` is not needed
* There is no cancel flow for this use case, and we only need `onDone`

Reply:
> This is all I changed, I suppose prettier did the above formatting change.

Can you revert that formatting change? Doesn't seem in line with code style.